src/librustc_privacy/lib.rs

```diff
@@ -710,7 +710,7 @@ impl<'a, 'tcx> PrivacyVisitor<'a, 'tcx> {
             NamedField(name) => format!("field `{}` of {} is private",
                                         name, struct_desc),
             UnnamedField(idx) => format!("field #{} of {} is private",
```

Comment:
This use of `#` always irks me - is it an american thing? I thought using '#' for 'number' went out of fashion in about 1900. The error message would be just as good without the '#', IMO.